x/concentrated-liquidity/internal/math/math_test.go

```diff
@@ -158,7 +158,7 @@ func (suite *ConcentratedMathTestSuite) TestCalcAmount0Delta() {
 			liquidity:       sdk.MustNewDecFromStr("1517882343.751510418088349649"), // we use the smaller liquidity between liq0 and liq1
 			sqrtPCurrent:    sdk.MustNewDecFromStr("70.710678118654752440"),         // 5000
 			sqrtPUpper:      sdk.MustNewDecFromStr("74.161984870956629487"),         // 5500
-			amount0Expected: "998976.618347426747968399",                            // TODO: should be 998976.618347426388356630
+			amount0Expected: "998976.618347426388356620",
```

Comment:
Why doesn't the wolfram result below match this?

Reply:
We are off from multiplying and then dividing the sdk.Dec. I can fix this if we want to transform the sqrtPriceA and sqrtPriceB into BigDecs when calculating 0 delta. What do you think?